actionlist/app/components/action_item_body.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Button,
+} from 'react-native';
+
+const styles = StyleSheet.create({
+  actionContainer: {
+    backgroundColor: '#990000',
+    padding: 5,
+    borderRadius: 8,
+    width: 100,
+  },
+  routeContainer: {
+    backgroundColor: '#ffffff',
+    padding: 5,
+    borderRadius: 8,
+    borderWidth: 1,
+    borderColor: '#990000',
+    marginRight: 10,
+    width: 100,
+  },
+  actionButton: {
+    fontSize: 15,
+    color: '#ffffff',
+  },
+  routeButton: {
+    fontSize: 15,
+    color: '#990000',
+  },
+});
+
+const routeLog = () => ({});
+
+const takeAction = () => ({});
+
+const ActionItemBody = rowData => (
```

Comment:
I dont know if we can do that. I tried spreading it in the params, but it requires that i use rowData. for every key